.github/workflows/test.yml

```diff
@@ -8,10 +8,21 @@ on:
   schedule:
     - cron:  '0 4 * * *'
 
+env:
+  DRUPAL_TESTING_TEST_CODING_STYLES: false
+  DRUPAL_TESTING_DATABASE_USER: root
+  DRUPAL_TESTING_DATABASE_PASSWORD: root
+  DRUPAL_TESTING_DATABASE_ENGINE: mysql
+
 jobs:
   build:
 
     runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        DRUPAL_TESTING_DRUPAL_VERSION: ['~8.8.0', '~9.0.0']
```

Comment:
What about testing 8.7 since it states `core_version_requirement: ^8.7.7 || ^9` below?